java/src/main/java/com/logicalclocks/hsfs/engine/hudi/DeltaStreamerTransformer.java

```diff
@@ -30,11 +26,6 @@
 import org.apache.spark.sql.SparkSession;
 
 public class DeltaStreamerTransformer implements Transformer {
-  private FeatureStore featureStore;
```

Comment:
What happened here?

Reply:
Some of the code was left over and was not used at all. So I removed them